bin/prelude.h

```diff
@@ -215,6 +218,12 @@ predicate ints(int *p, int count; list<int> vs) =
     :
         integer(p, ?v) &*& ints(p + 1, count - 1, ?vs0) &*& vs == cons(v, vs0);
 
+predicate llongs(long long *p, long long count; list<long long> ls) = 

```

Comment:
Not that it matters (because all integral types mean the same thing inside annotations) but for consistency, it seems better to just use `int` as the type of `count`. (And in any case, the size of an array is orthogonal to the size of its elements.)